src/types/index.ts

```diff
@@ -1159,7 +1159,7 @@ export interface CheckpointWithData {
   totalSupply: BigNumber;
 }
 
-export interface SecondaryKey {
+export interface SigningKey {
   signer: Signer;
```

Comment:
To me I would have though `Account` type should be sufficient here since I don't think we'd get an `Identity` from the related methods, and `Account` should be accepted by methods wanting the `Signer` since its a part of the union type of `Signer`.

I tried this change and it breaks some tests and is going to be more work. I am going to ask about it in Slack though.

Reply:
I looked at the card and it said this should become an `Account`

https://polymath.atlassian.net/browse/MSDK-760

So this is actually in scope for this PR.